app/assets/stylesheets/common/base/category-list.scss

```diff
@@ -340,3 +344,38 @@
 .category-box.no-category-boxes-style {
   border-left-width: 2px;
 }
+
+.muted-categories {
+  > a {
+    color: $primary-medium;
+    border: 1px solid $primary-low;
+    display: flex;
+    justify-content: space-between;
+    padding: 0.75em;
+    border-left-width: 6px;
+    border-right: none 0;
+    margin: 0 0 1em -3px;
```

Comment:
don't like much magic numbers in css, can we avoid this "-3px" ?

Reply:
I spent an hour to avoid it. But I'm unable to align the left border properly without it :disappointed: